packages/network-controller/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.2.0]
+### Changed
+- Update @metamask/eth-json-rpc-middleware in network controller ([#1988](https://github.com/MetaMask/core/pull/1988))
+- move json-rpc-engine from merged-packages to packages ([#1895](https://github.com/MetaMask/core/pull/1895))
```

Comment:
The impact on the network controller here is "Bump dependency on @metamask/json-rpc-engine to ^7.2.0", as documented in the PR

```suggestion
- Bump dependency on `@metamask/json-rpc-engine` to ^7.2.0 ([#1895](https://github.com/MetaMask/core/pull/1895))
```